squawk/data/preprocessing/augment.py

```diff
@@ -32,59 +136,79 @@ class SpecAugmentConfig(object):
     use_timewarp: bool = True
 
 
-class StandardAudioTransform(nn.Module):
+class StandardAudioTransform(AugmentModule):
 
```

Comment:
For every Transform, I think it would be great to add a brief description of what each one does.
Timestretch and Timeshift are clear but the other ones are hard to tell unless you actually read the augment function

Reply:
Duly noted.